pkgs/development/compilers/gerbil/build.nix

```diff
@@ -1,34 +1,51 @@
-{ stdenv, makeStaticLibraries,
+{ gccStdenv, lib, makeStaticLibraries, callPackage,
   coreutils, rsync, bash,
   openssl, zlib, sqlite, libxml2, libyaml, libmysqlclient, lmdb, leveldb, postgresql,
   version, git-version, gambit, src }:
 
+# We use Gambit, that works 10x better with GCC than Clang. See ../gambit/build.nix
+let stdenv = gccStdenv; in
+
 stdenv.mkDerivation rec {
   pname = "gerbil";
   inherit version;
   inherit src;
 
-  # Use makeStaticLibraries to enable creation of statically linked binaries
   buildInputs_libraries = [ openssl zlib sqlite libxml2 libyaml libmysqlclient lmdb leveldb postgresql ];
-  buildInputs_staticLibraries = map makeStaticLibraries buildInputs_libraries;
+  # Use makeStaticLibraries to enable creation of statically linked binaries
+  #buildInputs_staticLibraries = map makeStaticLibraries buildInputs_libraries;
```

Comment:
If it will no longer be needed, I guess it could be removed, and otherwise it's probably a good idea to write in the comment in which case it will come back.